src/components/header/SlidingNavMenu.jsx

```diff
@@ -11,7 +11,7 @@ class SlidingNavMenu extends React.PureComponent {
     const pagePathArray = Object.keys(pageOptions);
 
     this.state = {
-      activeKey: pagePathArray[0],
+      activeKey: pathname,
```

Comment:
@mogoodrich current sliding navigation always set's the first navItem as active, if it was not clicked. updated it to use the current location in the URL.